ui/app/pages/create-account/new-account.component.js

```diff
@@ -49,31 +49,31 @@ export default class NewAccountCreateForm extends Component {
         <div className="new-account-create-form__input-label">
           {this.context.t('accountName')}
         </div>
-        <div className="new-account-create-form__input-wrapper">
+        <div className="new-account-create-form__account-name">
```

Comment:
Does this class exist? I can't find it

Reply:
I can't find the old one either 😕

Reply:
Neither the old nor the new class exist, but the div wrapping of both the input and the button solves the problem. Delete the class entirely?

Reply:
>Delete the class entirely?

Sure, that sounds like it would be best